### ISSUE TITLE
Hi, regarding the nn1 of ViTResNet:

### ISSUE THREAD

**Issue by sdahan12** [tahmid0007/VisualTransformers#2]
Hi, regarding the nn1 of ViTResNet
shouldn't the dim be:

nn.Linear(dim, num_classes)

and not nn.Linear(dim, mlp_dim)?

because we want to do the classification

**Comment by tahmid0007** [comment]
That's right. It's been updated. Thanks.

**Comment by sdahan12** [comment]
Great happy to help
do you have any outputs from the training phase? , loss, accuracy?

**Comment by tahmid0007** [comment]
I dont have a log at this moment. Accuracy on CIFAR10 does not exceed ~80% as Transformers are best suited for larger datasets.
